packages/gestalt/src/formatErrorId.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+export default function formatErrorId(id: string): string {
```

Comment:
the overhead and the indirection of this helper method in my opinion is a bit confusing, I would prefer to just keep the string inline

Reply:
True. I abstracted it out so that if we decide to change it, we only have to change it in one place instead of many.

Reply:
It's probably also not to expose the word "gestalt" here - it'll show up in anchors etc. It's probably better served localized to a component or explicitly passed in by the user.